web/pom.xml

```diff
@@ -45,7 +45,34 @@
 
   <build>
     <finalName>oncotree</finalName>
+    <filters>
+        <filter>src/main/resources/application.properties</filter>
+    </filters>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+        <includes>
+            <include>**/application.properties</include>
+            <include>**/ehcache.xml</include>
+            <include>**/ehcache_backup.xml</include>
+            <include>**/log4j.properties</include>
+            <include>**/static/</include>
+         </includes>
+      </resource>
+    </resources>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>2.6</version>
+        <configuration>
+          <delimiters>
+            <delimiter>${*}</delimiter>
```

Comment:
I'm not familiar with this delimiter function. I think @averyniceday found a problem related to this pattern